integration_tests/testdata/operator/base/deployment-template.yaml

```diff
@@ -18,17 +18,30 @@ spec:
           image: {{.OperatorImage}}
           command:
             - spinnaker-operator
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Since we are loading the operator image within kind we are not needing to reload it.